homeassistant/components/rachio/strings.json

```diff
@@ -63,6 +63,16 @@
         }
       }
     },
+    "start_single_entity": {
```

Comment:
Maybe start enough, It's also possible they might want to target three different hose timers

Reply:
I wonder if we don't need this if they're implemented as valve entities

Reply:
As opposed to switch entities?

Reply:
Maybe just "start watering" since it can be used for timers and zones? We already have stop, pause and resume watering, and the start multiple zones schedule service is fairly self explanatory from the title.  

It also can start schedules if we make the duration optional, which would probably be a good idea, then it works on all the switch types.

Reply:
Sounds good to me